Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerUserStore.cs

```diff
@@ -113,7 +113,7 @@ public SqlServerUserStore(IAuthorizationDbContext authorizationDbContext)
 
         private static string[] SplitId(string id)
         {
-            var delimiter = new [] {@"\"};
+            var delimiter = new [] {@":"};
```

Comment:
Should we throw an ArgumentException here if there aren't two parts to the id?

Reply:
Yes, good idea.